[2024-08-19T23:10:19.309+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-19T23:10:19.504+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [queued]>
[2024-08-19T23:10:19.523+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [queued]>
[2024-08-19T23:10:19.524+0700] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-08-19T23:10:19.552+0700] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): modify_data> on 2024-08-18 00:00:00+00:00
[2024-08-19T23:10:19.559+0700] {standard_task_runner.py:64} INFO - Started process 34810 to run task
[2024-08-19T23:10:19.564+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'scrap_to_dag', 'modify_data', 'scheduled__2024-08-18T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/oto/scrap_oto.py', '--cfg-path', '/tmp/tmpkk0pbe7j']
[2024-08-19T23:10:19.567+0700] {standard_task_runner.py:91} INFO - Job 46: Subtask modify_data
[2024-08-19T23:10:19.661+0700] {task_command.py:426} INFO - Running <TaskInstance: scrap_to_dag.modify_data scheduled__2024-08-18T00:00:00+00:00 [running]> on host BrightHorizons.
[2024-08-19T23:10:19.824+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrap_to_dag' AIRFLOW_CTX_TASK_ID='modify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-18T00:00:00+00:00'
[2024-08-19T23:10:19.825+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-19T23:10:19.862+0700] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-08-19T23:10:19.864+0700] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py
[2024-08-19T23:10:22.075+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:22 WARN Utils: Your hostname, BrightHorizons resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2024-08-19T23:10:22.079+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:22 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-08-19T23:10:23.649+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SparkContext: Running Spark version 3.5.1
[2024-08-19T23:10:23.649+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-08-19T23:10:23.650+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SparkContext: Java version 11.0.24
[2024-08-19T23:10:23.743+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-08-19T23:10:23.843+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceUtils: ==============================================================
[2024-08-19T23:10:23.843+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-08-19T23:10:23.844+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceUtils: ==============================================================
[2024-08-19T23:10:23.844+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SparkContext: Submitted application: CarDataAnalysis
[2024-08-19T23:10:23.874+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-08-19T23:10:23.894+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-08-19T23:10:23.897+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-08-19T23:10:23.992+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SecurityManager: Changing view acls to: root
[2024-08-19T23:10:23.993+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SecurityManager: Changing modify acls to: root
[2024-08-19T23:10:23.994+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SecurityManager: Changing view acls groups to:
[2024-08-19T23:10:23.995+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SecurityManager: Changing modify acls groups to:
[2024-08-19T23:10:23.996+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:23 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: root; groups with view permissions: EMPTY; users with modify permissions: root; groups with modify permissions: EMPTY
[2024-08-19T23:10:24.423+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO Utils: Successfully started service 'sparkDriver' on port 46027.
[2024-08-19T23:10:24.479+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO SparkEnv: Registering MapOutputTracker
[2024-08-19T23:10:24.544+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO SparkEnv: Registering BlockManagerMaster
[2024-08-19T23:10:24.577+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-08-19T23:10:24.578+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-08-19T23:10:24.635+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-08-19T23:10:24.671+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-b3bd5987-b8da-4fe2-b383-e469c7ce1247
[2024-08-19T23:10:24.702+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-08-19T23:10:24.762+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:24 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-08-19T23:10:25.010+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:25 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-08-19T23:10:25.097+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:25 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-08-19T23:10:25.364+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:25 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-08-19T23:10:26.996+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:27.998+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:29.000+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:30.002+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:31.004+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:32.006+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:33.008+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:34.010+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:35.011+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:36.013+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:37.030+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:38.032+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:39.034+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:40.035+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:41.037+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:42.038+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:43.040+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:44.042+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:45.044+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:46.046+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:10:46.048+0700] {spark_submit.py:579} INFO - 24/08/19 23:10:46 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 28049ms.
[2024-08-19T23:11:15.100+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:16.102+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:17.103+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:18.105+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:19.107+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:20.110+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:21.112+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:22.113+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:23.114+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:24.116+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:24.118+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:24 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 20954ms.
[2024-08-19T23:11:46.075+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:47.077+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:48.079+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:49.080+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:50.082+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:51.084+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:52.086+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:53.087+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:54.089+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:55.091+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:11:55.093+0700] {spark_submit.py:579} INFO - 24/08/19 23:11:55 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 38480ms.
[2024-08-19T23:12:34.576+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:35.577+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:36.578+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:37.580+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:38.581+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:39.583+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:40.585+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:41.587+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:42.588+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:43.590+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:12:43.592+0700] {spark_submit.py:579} INFO - 24/08/19 23:12:43 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 23449ms.
[2024-08-19T23:13:08.043+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:09.044+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:10.046+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:11.048+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:12.050+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:13.052+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:14.054+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:15.056+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:16.058+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:17.060+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:17.062+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:17 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 37073ms.
[2024-08-19T23:13:55.138+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:56.139+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:57.141+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:58.142+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:13:59.144+0700] {spark_submit.py:579} INFO - 24/08/19 23:13:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:00.146+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:01.148+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:02.150+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:03.152+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:04.154+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:04.155+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:04 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 6 failover attempts. Trying to failover after sleeping for 40013ms.
[2024-08-19T23:14:45.172+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:46.173+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:47.175+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:48.178+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:49.180+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:50.183+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:51.184+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:52.185+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:53.187+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:54.188+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:14:54.190+0700] {spark_submit.py:579} INFO - 24/08/19 23:14:54 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 7 failover attempts. Trying to failover after sleeping for 32081ms.
[2024-08-19T23:15:27.274+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:28.276+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:29.278+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:30.280+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:31.282+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:32.285+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:33.286+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:34.288+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:35.289+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:36.291+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:15:36.293+0700] {spark_submit.py:579} INFO - 24/08/19 23:15:36 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 8 failover attempts. Trying to failover after sleeping for 30792ms.
[2024-08-19T23:16:08.089+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:09.090+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:10.093+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:11.094+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:12.096+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:13.097+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:14.099+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:15.101+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:16.103+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:17.105+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:17.108+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:17 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 9 failover attempts. Trying to failover after sleeping for 33646ms.
[2024-08-19T23:16:51.757+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:52.759+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:53.760+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:54.761+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:55.763+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:56.765+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:57.767+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:58.769+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:16:59.770+0700] {spark_submit.py:579} INFO - 24/08/19 23:16:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:00.772+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:00.774+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:00 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 10 failover attempts. Trying to failover after sleeping for 36020ms.
[2024-08-19T23:17:37.798+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:38.800+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:39.802+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:40.804+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:41.805+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:42.807+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:43.809+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:44.811+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:45.812+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:46.814+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:17:46.816+0700] {spark_submit.py:579} INFO - 24/08/19 23:17:46 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 11 failover attempts. Trying to failover after sleeping for 24607ms.
[2024-08-19T23:18:12.426+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:13.427+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:14.429+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:15.430+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:16.432+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:17.435+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:18.437+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:19.438+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:20.440+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:21.441+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:21.443+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:21 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 12 failover attempts. Trying to failover after sleeping for 36052ms.
[2024-08-19T23:18:58.498+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:18:59.500+0700] {spark_submit.py:579} INFO - 24/08/19 23:18:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:00.501+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:01.503+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:02.504+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:03.506+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:04.508+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:05.509+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:06.511+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:07.513+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:07.514+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:07 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 13 failover attempts. Trying to failover after sleeping for 38185ms.
[2024-08-19T23:19:46.702+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:47.704+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:48.705+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:49.706+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:50.708+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:51.710+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:52.712+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:53.713+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:54.714+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:55.716+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:19:55.718+0700] {spark_submit.py:579} INFO - 24/08/19 23:19:55 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 14 failover attempts. Trying to failover after sleeping for 18358ms.
[2024-08-19T23:20:15.079+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:16.081+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:17.082+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:18.083+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:19.085+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:20.086+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:21.088+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:22.089+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:23.091+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:24.093+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:20:24.095+0700] {spark_submit.py:579} INFO - 24/08/19 23:20:24 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 15 failover attempts. Trying to failover after sleeping for 44988ms.
[2024-08-19T23:21:10.087+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:11.088+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:12.090+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:13.090+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:14.091+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:15.093+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:16.094+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:17.096+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:18.097+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:19.098+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:19.101+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:19 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 16 failover attempts. Trying to failover after sleeping for 32241ms.
[2024-08-19T23:21:52.344+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:53.346+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:54.347+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:55.348+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:56.350+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:57.352+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:58.354+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:21:59.355+0700] {spark_submit.py:579} INFO - 24/08/19 23:21:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:00.356+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:01.358+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:01.360+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:01 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 17 failover attempts. Trying to failover after sleeping for 40419ms.
[2024-08-19T23:22:42.782+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:43.783+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:44.785+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:45.787+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:46.789+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:47.791+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:48.792+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:49.794+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:50.795+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:51.797+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:22:51.798+0700] {spark_submit.py:579} INFO - 24/08/19 23:22:51 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 18 failover attempts. Trying to failover after sleeping for 40478ms.
[2024-08-19T23:23:33.281+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:34.282+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:35.284+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:36.285+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:37.287+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:38.288+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:39.290+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:40.291+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:41.294+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:42.296+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:23:42.299+0700] {spark_submit.py:579} INFO - 24/08/19 23:23:42 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 19 failover attempts. Trying to failover after sleeping for 28091ms.
[2024-08-19T23:24:11.394+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:12.396+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:13.398+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:14.399+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:15.401+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:16.403+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:17.406+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:18.408+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:19.410+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:20.411+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:20.413+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:20 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 20 failover attempts. Trying to failover after sleeping for 25762ms.
[2024-08-19T23:24:47.178+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:48.179+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:49.181+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:50.183+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:51.184+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:52.185+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:53.186+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:54.188+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:55.189+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:56.191+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:24:56.193+0700] {spark_submit.py:579} INFO - 24/08/19 23:24:56 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 21 failover attempts. Trying to failover after sleeping for 31843ms.
[2024-08-19T23:25:29.039+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:30.040+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:31.041+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:32.043+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:33.045+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:34.046+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:35.047+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:36.049+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:37.051+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:38.052+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:38.056+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:38 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 22 failover attempts. Trying to failover after sleeping for 16775ms.
[2024-08-19T23:25:55.835+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:56.836+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:57.838+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:58.840+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:25:59.841+0700] {spark_submit.py:579} INFO - 24/08/19 23:25:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:00.842+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:01.843+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:02.845+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:03.847+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:04.849+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:04.851+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:04 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 23 failover attempts. Trying to failover after sleeping for 42444ms.
[2024-08-19T23:26:48.297+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:49.298+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:50.300+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:51.302+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:52.303+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:53.304+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:54.306+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:55.307+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:56.309+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:57.310+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:26:57.311+0700] {spark_submit.py:579} INFO - 24/08/19 23:26:57 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 24 failover attempts. Trying to failover after sleeping for 32797ms.
[2024-08-19T23:27:31.110+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:32.112+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:33.113+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:34.115+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:35.118+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:36.119+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:37.120+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:38.122+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:39.123+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:40.124+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:27:40.126+0700] {spark_submit.py:579} INFO - 24/08/19 23:27:40 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 25 failover attempts. Trying to failover after sleeping for 30160ms.
[2024-08-19T23:28:11.289+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:12.290+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:13.292+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:14.294+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:15.296+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:16.298+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:17.299+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:18.300+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:19.301+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:20.303+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:20.305+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:20 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 26 failover attempts. Trying to failover after sleeping for 35915ms.
[2024-08-19T23:28:57.222+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:58.223+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:28:59.225+0700] {spark_submit.py:579} INFO - 24/08/19 23:28:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:00.227+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:01.228+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:02.229+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:03.231+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:03 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:04.232+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:05.234+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:06.235+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:06.236+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:06 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 27 failover attempts. Trying to failover after sleeping for 29059ms.
[2024-08-19T23:29:36.298+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:37.300+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:38.301+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:39.302+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:40.303+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:41.304+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:42.306+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:43.308+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:44.309+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:45.311+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:29:45.313+0700] {spark_submit.py:579} INFO - 24/08/19 23:29:45 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 28 failover attempts. Trying to failover after sleeping for 32275ms.
[2024-08-19T23:30:18.591+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:19.592+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:20.593+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:21.594+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:22.596+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:23.598+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:24.599+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:25.600+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:26.602+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:27.604+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:27.606+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:27 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 29 failover attempts. Trying to failover after sleeping for 25294ms.
[2024-08-19T23:30:53.902+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:54.904+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:55.905+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:56.906+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:57.908+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:58.909+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:30:59.911+0700] {spark_submit.py:579} INFO - 24/08/19 23:30:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:31:00.912+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:31:01.914+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:31:02.915+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-08-19T23:31:02.921+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 ERROR SparkContext: Error initializing SparkContext.
[2024-08-19T23:31:02.922+0700] {spark_submit.py:579} INFO - java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort
[2024-08-19T23:31:02.922+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.GeneratedConstructorAccessor28.newInstance(Unknown Source)
[2024-08-19T23:31:02.922+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:31:02.923+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:31:02.923+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-08-19T23:31:02.924+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:824)
[2024-08-19T23:31:02.924+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-08-19T23:31:02.925+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-08-19T23:31:02.925+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-08-19T23:31:02.925+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-08-19T23:31:02.926+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-08-19T23:31:02.926+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy29.getNewApplication(Unknown Source)
[2024-08-19T23:31:02.927+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.api.impl.pb.client.ApplicationClientProtocolPBClientImpl.getNewApplication(ApplicationClientProtocolPBClientImpl.java:286)
[2024-08-19T23:31:02.927+0700] {spark_submit.py:579} INFO - at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[2024-08-19T23:31:02.928+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-19T23:31:02.928+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-08-19T23:31:02.929+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-08-19T23:31:02.930+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-08-19T23:31:02.930+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-08-19T23:31:02.930+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-08-19T23:31:02.931+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-08-19T23:31:02.931+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy30.getNewApplication(Unknown Source)
[2024-08-19T23:31:02.931+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.getNewApplication(YarnClientImpl.java:284)
[2024-08-19T23:31:02.932+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.createApplication(YarnClientImpl.java:292)
[2024-08-19T23:31:02.932+0700] {spark_submit.py:579} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:195)
[2024-08-19T23:31:02.932+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2024-08-19T23:31:02.932+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2024-08-19T23:31:02.932+0700] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2024-08-19T23:31:02.933+0700] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-19T23:31:02.933+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-19T23:31:02.934+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2024-08-19T23:31:02.934+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:31:02.936+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:31:02.936+0700] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-19T23:31:02.936+0700] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-19T23:31:02.937+0700] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-08-19T23:31:02.938+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-08-19T23:31:02.939+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-08-19T23:31:02.939+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-08-19T23:31:02.940+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-08-19T23:31:02.940+0700] {spark_submit.py:579} INFO - ... 33 more
[2024-08-19T23:31:02.940+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-08-19T23:31:02.959+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 INFO SparkUI: Stopped Spark web UI at http://10.255.255.254:4040
[2024-08-19T23:31:02.973+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to send shutdown message before the AM has registered!
[2024-08-19T23:31:02.984+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 WARN YarnSchedulerBackend$YarnSchedulerEndpoint: Attempted to request executors before the AM has registered!
[2024-08-19T23:31:02.991+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:02 INFO YarnClientSchedulerBackend: Shutting down all executors
[2024-08-19T23:31:03.000+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO YarnSchedulerBackend$YarnDriverEndpoint: Asking each executor to shut down
[2024-08-19T23:31:03.008+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO YarnClientSchedulerBackend: YARN client scheduler backend Stopped
[2024-08-19T23:31:03.047+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-08-19T23:31:03.064+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO MemoryStore: MemoryStore cleared
[2024-08-19T23:31:03.065+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO BlockManager: BlockManager stopped
[2024-08-19T23:31:03.085+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-08-19T23:31:03.087+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 WARN MetricsSystem: Stopping a MetricsSystem that is not running
[2024-08-19T23:31:03.091+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-08-19T23:31:03.108+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO SparkContext: Successfully stopped SparkContext
[2024-08-19T23:31:03.150+0700] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-08-19T23:31:03.151+0700] {spark_submit.py:579} INFO - File "/home/miracle/mobil/spark/oto.py", line 11, in <module>
[2024-08-19T23:31:03.152+0700] {spark_submit.py:579} INFO - spark = SparkSession.builder.appName("CarDataAnalysis").getOrCreate()
[2024-08-19T23:31:03.153+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 497, in getOrCreate
[2024-08-19T23:31:03.154+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 515, in getOrCreate
[2024-08-19T23:31:03.155+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 203, in __init__
[2024-08-19T23:31:03.156+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 296, in _do_init
[2024-08-19T23:31:03.157+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 421, in _initialize_context
[2024-08-19T23:31:03.158+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1587, in __call__
[2024-08-19T23:31:03.159+0700] {spark_submit.py:579} INFO - File "/usr/local/lib/python3.10/dist-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-08-19T23:31:03.159+0700] {spark_submit.py:579} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2024-08-19T23:31:03.160+0700] {spark_submit.py:579} INFO - : java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort
[2024-08-19T23:31:03.160+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.GeneratedConstructorAccessor28.newInstance(Unknown Source)
[2024-08-19T23:31:03.160+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:31:03.161+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:31:03.161+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-08-19T23:31:03.162+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:824)
[2024-08-19T23:31:03.162+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-08-19T23:31:03.162+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-08-19T23:31:03.163+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-08-19T23:31:03.163+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-08-19T23:31:03.163+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-08-19T23:31:03.164+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy29.getNewApplication(Unknown Source)
[2024-08-19T23:31:03.164+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.api.impl.pb.client.ApplicationClientProtocolPBClientImpl.getNewApplication(ApplicationClientProtocolPBClientImpl.java:286)
[2024-08-19T23:31:03.164+0700] {spark_submit.py:579} INFO - at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
[2024-08-19T23:31:03.165+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-19T23:31:03.165+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-08-19T23:31:03.166+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-08-19T23:31:03.166+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-08-19T23:31:03.166+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-08-19T23:31:03.167+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-08-19T23:31:03.167+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-08-19T23:31:03.167+0700] {spark_submit.py:579} INFO - at com.sun.proxy.$Proxy30.getNewApplication(Unknown Source)
[2024-08-19T23:31:03.168+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.getNewApplication(YarnClientImpl.java:284)
[2024-08-19T23:31:03.170+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.yarn.client.api.impl.YarnClientImpl.createApplication(YarnClientImpl.java:292)
[2024-08-19T23:31:03.171+0700] {spark_submit.py:579} INFO - at org.apache.spark.deploy.yarn.Client.submitApplication(Client.scala:195)
[2024-08-19T23:31:03.171+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.cluster.YarnClientSchedulerBackend.start(YarnClientSchedulerBackend.scala:62)
[2024-08-19T23:31:03.171+0700] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.TaskSchedulerImpl.start(TaskSchedulerImpl.scala:235)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:604)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-19T23:31:03.172+0700] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
[2024-08-19T23:31:03.173+0700] {spark_submit.py:579} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:777)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-08-19T23:31:03.174+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-08-19T23:31:03.175+0700] {spark_submit.py:579} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-08-19T23:31:03.175+0700] {spark_submit.py:579} INFO - ... 33 more
[2024-08-19T23:31:03.175+0700] {spark_submit.py:579} INFO - 
[2024-08-19T23:31:03.398+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO ShutdownHookManager: Shutdown hook called
[2024-08-19T23:31:03.400+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO ShutdownHookManager: Deleting directory /tmp/spark-01191503-1e5a-4535-b64a-1d620c0f3503
[2024-08-19T23:31:03.410+0700] {spark_submit.py:579} INFO - 24/08/19 23:31:03 INFO ShutdownHookManager: Deleting directory /tmp/spark-e7305aca-76e3-4bcf-9420-7a063899725e
[2024-08-19T23:31:03.464+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-19T23:31:03.486+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py. Error code is: 1.
[2024-08-19T23:31:03.493+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=scrap_to_dag, task_id=modify_data, run_id=scheduled__2024-08-18T00:00:00+00:00, execution_date=20240818T000000, start_date=20240819T161019, end_date=20240819T163103
[2024-08-19T23:31:03.517+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 46 for task modify_data (Cannot execute: spark-submit --master yarn --name arrow-spark --queue root.default /home/miracle/mobil/spark/oto.py. Error code is: 1.; 34810)
[2024-08-19T23:31:03.563+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-19T23:31:03.597+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-19T23:31:03.600+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
